qfieldsync/core/cloud_api.py

```diff
@@ -114,13 +122,73 @@ def from_reply(reply: QNetworkReply) -> Optional[CloudException]:
     return CloudException(reply, Exception(message))
 
 
+class CloudAuthMethod(Enum):
+    NONE = 0
+    CREDENTIALS = 1
+    SSO = 2
+
+
+def build_oauth2_auth_config(
+    auth_data: dict[str, Any],
+    related_uri: str,
+    config_name: str = "qfieldcloud_sso",
+    should_persist: bool = True,
```

Comment:
```suggestion
    should_persist_token: bool = True,
```
or
```suggestion
    is_token_persisted: bool = True,
```
I think it is better?

Reply:
I personally prefer the `should` since we just pass it to the QGIS auth manager.